doc/dbus_api.md

```diff
@@ -111,95 +111,95 @@ Iface: o.o.YaST.Installer1.Software
 -  PropertiesChanged ( only standard one from org.freedesktop.DBus.Properties interface )
 
 
-## `org.opensuse.DInstaller.Storage` Service
+## `org.opensuse.Agama.Storage1` Service
 
 Service for managing storage devices.
 
 ### Overview
 
 ~~~
-/DInstaller/Storage1
+/Agama/Storage1
   .ObjectManager
-  .DInstaller.ServiceStatus1
-  .DInstaller.Progress1
-  .DInstaller.Validation1
-  .DInstaller.Storage1
-  .DInstaller.Storage1.Proposal.Calculator
-  .DInstaller.Storage1.ISCSI.Initiator
-  .DInstaller.Storage1.DASD.Manager (Only available on s390 systems)
-/DInstaller/Storage1/Proposal
-  .DInstaller.Storage1.Proposal
-/DInstaller/Storage1/iscsi_nodes/[0-9]+
-  .DInstaller.Storage1.ISCSI.Node
-/DInstaller/Storage1/dasds/[0-9]+ (Only available on s390 systems)
-  .DInstaller.Storage1.DASD.device
-/DInstaller/Storage1/jobs/[0-9]+
-  .DInstaller.Storage1.Job
-  .DInstaller.Storage1.DASD.Format
+  .Agama1.ServiceStatus
+  .Agama1.Progress
+  .Agama1.Validation
```

Comment:
also these 3, I believe we want `.Agama.Foo1`

Reply:
Not really, the idea is to have the version number in the service name.